test_runner/fixtures/utils.py

```diff
@@ -33,9 +33,20 @@
     )
 
     from fixtures.neon_fixtures import PgBin
+from collections import OrderedDict
+
+from allpairspy import AllPairs
+
```

Comment:
The order of imports became weird after the latest merge.

- `from collections import OrderedDict` should go to the top block of system imports
- `from allpairspy import AllPairs` should to the second block with imports from dependencies
- also, if you could move the line below (`from fixtures.common_types import TimelineId`) to the third block of local imports  that would be nice

Reply:
Fixed, thanks